### DISCUSSION TITLE
Postgres select table with schema

### DISCUSSION THREAD

**Discussion by AdamJSoftware** [SeaQL/sea-query#523] [Q&A] [answered]
Just want to add a schema to when I select a table from my database. Tried looking through the issues and I found something similar but nothing with a clear solution. So documenting this would probably also be extremely useful.

```
ex: SELECT * FROM priv_schema.table_whatever;
```

**Comment by AdamJSoftware** [comment]
I've tried using the iden macro in order to give it a custom name,  however the quotes cause a problem
```

#[derive(Iden)]
pub enum OrganizationTable {
    // #[iden = "auto"]
    // Schema,
    #[iden = "auto.ppm_organization"]
    Table,
    Name,
    Description,
    LatestVersion,
    Organization,
    Tags
}	
```
produces: 
```
SELECT * FROM \"auto.ppm_organization\"
```
Is there somehow a way to get around this?

**Comment by AdamJSoftware** [comment, marked as answer]
Figured it out. Documentation is extremely lacking for this. Finally found it after skimming through this pull request which I found while reading the patch logs. https://github.com/SeaQL/sea-schema/pull/26
```
#[derive(Iden)]
pub enum OrganizationTable {
    // #[iden = "auto"]
    // Schema,
    #[iden = "auto.ppm_organization"]
    Table,
    Name,
    Description,
    LatestVersion,
    Organization,
    Tags
}	
  let (query, args) = Query::select().from((OrganizationTable::Schema, OrganizationTable::Table)).columns([Plugin::Name]).build_postgres(PostgresQueryBuilder);

```

**Reply by billy1624** [reply]
Hey @AdamJSoftware, thanks for the updates!! Do you think we can add this to somewhere in the documentation?
